packages/patternfly-4/react-table/src/components/Table/Table.md

```diff
@@ -118,6 +114,55 @@ class SimpleTable extends React.Component {
 }
 ```
 
+## Table with custom row wrapper
+
+```js
+import React from 'react';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+} from '@patternfly/react-table';
+
+import { CustomRowWrapper } from './demo/CustomRowWrapper';
```

Comment:
As a user, I want to see the CustomRowWrapper code. Can we please include it in the SimpleTable class?

Reply:
Totally agree, but it doesn't seem like our docs allow more than one class to be present in a single example area

<img width="484" alt="Screen Shot 2019-10-18 at 9 12 16 AM" src="https://user-images.githubusercontent.com/5942899/67097479-334c5e80-f188-11e9-8f71-7635f34cdb9f.png">

I added a comment that will point to where the implementation lives (once it's merged) so users are not forced to hunt around for it. Think this is an ok stop-gap until we get support for multiple classes in a single example snippet?

<img width="1220" alt="Screen Shot 2019-10-18 at 9 17 00 AM" src="https://user-images.githubusercontent.com/5942899/67097573-5d058580-f188-11e9-94c5-c00f57eb9c5a.png">

We could possibly introduce custom decorator support to help here, something like `@noparse`. TBH though I'd much prefer we go back to using proper javascript or typescript for these examples instead because these markdown files continue to require more custom code to make them act like normal script files again. WDYT? Can we go with the simple comment for now?

Reply:
Btw [the url in that comment](https://github.com/patternfly/patternfly-react/pull/3160/files#diff-f6c0274c6d946d8f7472804d7595cf78R129) will only work once this PR is merged. In the mean time you can view what it would contain on [my working branch](https://github.com/seanforyou23/patternfly-react/blob/row-wrapper-demo/packages/patternfly-4/react-table/src/components/Table/demo/CustomRowWrapper.js).

Reply:
Considering CustomRowWrapper.js does not contain any state, do you need a class? Could you add something similar (below) in your constructor?

```
this.customRowWrapper = ({
  trRef,
  className,
  rowProps,
  row: { isExpanded, isHeightAuto }
}) => {
  const isEvenOddRow = (rowProps.rowIndex + 1) % 2;
  const customStyle = {
    borderLeft: '3px solid var(--pf-global--primary-color--100)'
  };
  return (
    ...
  );
}
```

Reply:
I ran into the same situation (not external variables) with the chart legend examples. I use the same approach (above) to create my custom legend and label components.